src/constant.rs

```diff
@@ -96,14 +107,18 @@ impl Iterator for ConstantBackoff {
     type Item = Duration;
 
     fn next(&mut self) -> Option<Self::Item> {
+        let delay = || match self.jitter {
+            true => self.delay + self.delay.mul_f32(fastrand::f32()),
```

Comment:
In default case it's a little strange to set delay(1s) + jitter(1s in some cases), so double the main delay. But maybe it's ok.

Reply:
We can add jitter range for user in the future, let's merge first.